spec.html

```diff
@@ -35867,7 +35867,7 @@ <h1>Runtime Semantics: InternalizeJSONProperty ( _holder_, _name_ )</h1>
 
     <emu-clause id="sec-json.stringify">
       <h1>JSON.stringify ( _value_ [ , _replacer_ [ , _space_ ] ] )</h1>
-      <p>The `stringify` function returns a String in UTF-16 encoded JSON format representing an ECMAScript value. It can take three parameters. The _value_ parameter is an ECMAScript value, which is usually an object or array, although it can also be a String, Boolean, Number or *null*. The optional _replacer_ parameter is either a function that alters the way objects and arrays are stringified, or an array of Strings and Numbers that acts as an inclusion list for selecting the object properties that will be stringified. The optional _space_ parameter is a String or Number that allows the result to have white space injected into it to improve human readability.</p>
+      <p>The `stringify` function returns a String in UTF-16 encoded JSON format representing an ECMAScript value, or undefined. It can take three parameters. The _value_ parameter is an ECMAScript value, which is usually an object or array, although it can also be a String, Boolean, Number or *null*. The optional _replacer_ parameter is either a function that alters the way objects and arrays are stringified, or an array of Strings and Numbers that acts as an inclusion list for selecting the object properties that will be stringified. The optional _space_ parameter is a String or Number that allows the result to have white space injected into it to improve human readability.</p>
```

Comment:
```suggestion
      <p>The `stringify` function returns a String in UTF-16 encoded JSON format representing an ECMAScript value, or *undefined*. It can take three parameters. The _value_ parameter is an ECMAScript value, which is usually an object or array, although it can also be a String, Boolean, Number or *null*. The optional _replacer_ parameter is either a function that alters the way objects and arrays are stringified, or an array of Strings and Numbers that acts as an inclusion list for selecting the object properties that will be stringified. The optional _space_ parameter is a String or Number that allows the result to have white space injected into it to improve human readability.</p>
```

Reply:
Separately, I’m a bit confused why object and array are lowercase, but String, Number, and Boolean are capitalized. @tc39/ecma262-editors?

Reply:
Lowercase o object is used to refer to anything that is an Object (https://tc39.github.io/ecma262/#sec-terms-and-definitions-object). Just an editorial convention, but one I like as Object is used frequently. Array should probably be capitalized however.